src/Console/ConsoleIo.php

```diff
@@ -300,7 +300,7 @@ public function success($message, int $newlines = 1, int $level = self::NORMAL):
      * @return void
      * @throws \Cake\Console\Exception\StopException
      */
-    public function abort($message, $code = CommandInterface::CODE_ERROR): void
+    public function abort(string $message, int $code = CommandInterface::CODE_ERROR): void
```

Comment:
`error()` accepts `array|string` and this is only `string` , !!?

Reply:
Looks like StopException expects a single string.